edx_proctoring/templates/proctored_exam/proctoring_opt_out_button.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}
+
+{% if allow_proctoring_opt_out %}
```

Comment:
@robrap I decided to include the test both here and in the parent templates. It seemed risky not to have it here, but it seemed unclear in the parent templates not to show that the include was conditional. Let me know if you'd rather I only do the test in one place or the other.